docs/tutorials/guides/mouse-input.mdx

```diff
@@ -0,0 +1,190 @@
+
+Create some class fields that can hold the state of the mouse and also the amount of how much we want to change
+the height of the window.
+
+```csharp
+public class Game : Window
+{
+	private const int HeightChangeAmount = 50;
+    private readonly IAppInput<MouseState> mouse;
+    private MouseState prevMouseState;
+}
+```
+
+### <span class="color-sub-step">Step 2.2: Creating a mouse object</span>
+
+To enable mouse input in your Velaptor game, you'll need to create a new instance of `IAppInput<MouseState>`,
+which will be set to the class field `mouse`. This will provide access to the current state of the mouse.
+You can create a new mouse object quickly and easily using Velaptor's `InputFactory` class.
+
+Use the factory to create the mouse object in the `Game` constructor.
+
+```csharp
+public Game() => this.mouse = InputFactory.CreateMouse();
+```
+
+
+## <span class="color-step">Step 3: Changing the window size</span>
+
+Perform the following steps to change the size of the window by using the mouse.
+
+### <span class="color-sub-step">Step 3.1: Getting the mouse state</span>
+
+We can get the state of the mouse by using the `mouse` object that we saved to the class field.  In the `OnUpdate()` method,
+use the `mouse` object to get the current state of the mouse.  Also, save the current mouse state as the previous mouse state.
+This way, when the next frame is executed, we can compare the current mouse state to the previous mouse state to see 
+if there are differences between the two states to detect if a button has been pressed or the mouse has moved.
+
+```csharp
+protected override void OnUpdate(FrameTime frameTime)
+{
+	var currMouseState = this.mouse.GetState();
+
+	this.prevMouseState = currMouseState;
+
+	base.OnUpdate(frameTime);
+}
+```
+
+### <span class="color-sub-step">Step 3.2: Changing the window width</span>
+
+Next, between the two mouse states in the `OnUpdate()` method, add the code below to change the width of the window.
+
+```csharp
+private void OnUpdate(FrameTime frameTime)
+{
+	... 
+
+	// If the left mouse button is currently being held in the down position
+	if (currMouseState.GetButtonState(MouseButton.LeftButton))
+	{
+		// The position of the mouse for the current frame
+		var currMousePos = currMouseState.GetPosition();
+
+		// The position of the mouse in the previous frame
+		var prevMousePos = this.prevMouseState.GetPosition();
+
+		// Add the mouse movement amount to the current width of the window
+		Width += (uint)(currMousePos.X - prevMousePos.X);
+	}
+
+	this.prevMouseState = currMouseState;
+
+	base.OnUpdate(frameTime);
+}
+```
+
+This code checks if the left mouse button is currently being held down. If it is, then the width of the window will be
+changed based on the difference between the current and previous mouse positions.  As you move the mouse left and right,
+it will take the difference between the position on the X-axis and apply that to the width of the window.
```

Comment:
Reword
```suggestion
This code checks whether  or not the left mouse button is currently being held down. If it is, then the width of the window will change based on the difference between the current and previous mouse positions.  As you move the mouse left and right, it will take the difference between the position on the X-axis and apply that to the width of the window.
```